src/packages.js

```diff
@@ -395,8 +396,12 @@ export default class Packages {
       const append = list
         .map(iter => ({
```

Comment:
I think maybe it's a good idea to split out this mapping into a utility function in `src/utils/packages.js`.

Reply:
Doing this will fix the codeclimate complaint as well :)